root/browse.html

```diff
@@ -7,6 +7,13 @@
 </div>
 
 <ul class="nav-list slidepanel">
+  <% IF maturity == 'developer' %>
+  <li>
+    <span class="dist-release maturity-developer">
+      <b class="release-name">Development release</b>
+    </span>
+  </li>
+  <% END %>
```

Comment:
It looks like the `<li>` and its contents don't change in the various use cases.  Could we use an include for that?

Reply:
That's a good idea. I'm on it.

Reply:
As requested, I made a new template in inc/ to reduce the redundancies.